test/ansible-inventory/molecule/test-local/playbook.yml

```diff
@@ -1,6 +1,6 @@
 ---
 
-- name: Build Operator in Kubernetes docker container
+- name: Build Operator in Kubernetes docker container for Molecule Test
```

Comment:
Same here

Reply:
However, when we are trying to follow the logs is hard to know from where the log came and has more than one ansible test with these names.

Reply:
I am closing it as solved since after our chat it shows make sense for you too.